models/snapchat.yml

```diff
@@ -246,4 +246,131 @@ models:
       - name: conversion_purchases_value
         description: The total value of purchases for an ad on the given day.
       - name: total_conversions
-        description: The total conversions being brought in the conversion fields you select in your `snapchat_ads__conversion_fields`.
\ No newline at end of file
+        description: The total conversions being brought in the conversion fields you select in your `snapchat_ads__conversion_fields`.
+
+  - name: snapchat_ads__campaign_country_report
+    description: Each record represents the daily performance of a Snapchat campaign per country.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - source_relation
+            - campaign_id
+            - country
+            - date_day
+    columns:
+      - name: source_relation
+        description: "{{ doc('source_relation') }}"
+      - name: date_day
+        description: The date of the report.
+        tests:
+          - not_null
+      - name: campaign_id
+        description: The ID of the campaign in Snapchat.
+        tests:
+          - not_null
+      - name: country
+        description: The country where the campaign was delivered.
+        tests:
+          - not_null
+      - name: ad_account_id
+        description: The ID of the account in Snapchat.
+      - name: ad_account_name
+        description: The name of the account in Snapchat.
+      - name: campaign_name
+        description: The name of the campaign in Snapchat.
+      - name: currency
+        description: The currency used by the account in Snapchat.
+      - name: spend
+        description: The spend on the campaign in the given day.
+      - name: impressions
+        description: The number of impressions the campaign had on the given day.
+      - name: swipes
+        description: The number of swipes the campaign had on the given day.
+      - name: shares
+        description: Number of times components of the campaign were shared.
+      - name: saves
+        description: Number of times components of the campaign were saved to Memories.
+      - name: total_conversions
+        description: The total conversions being brought in the conversion fields you select in your `snapchat_ads__conversion_fields`.
+      - name: conversion_purchases
```

Comment:
It looks like this field isn't brought into the end model, is this supposed to be brought in or should this be removed from the yml?

Reply:
see above